tuf/store/filestore.go

```diff
@@ -53,7 +55,14 @@ func (f *FilesystemStore) GetMeta(name string, size int64) ([]byte, error) {
 		}
 		return nil, err
 	}
-	return meta, nil
+	if size == -1 {
```

Comment:
Technically I think `strings.Replace` actually accepts any negative number for "unlimited" (https://golang.org/src/strings/strings.go?s=18112:18158#L681)  -- so we could change to any negative number or keep it as `-1`